.github/workflows/interface-unit-tests.yml

```diff
@@ -212,7 +261,8 @@ jobs:
            || fromJSON(needs.setup-ci-load.outputs.matrix-max-parallel).default
          }}
       matrix:
-        group: [1, 2, 3]
+        group: >-
```

Comment:
@rashidnhm 
Any idea how I can coerce GHA to accept this. The idea being:
* Splitting the tests into groups does not play nicely with the WAE workload, as I fail-fast doesn't propagate to the group boundaries.
* I need to have a single group for the WAE runs to ensure every test is hit.
* I am attempting this by passing this data with a new step-set variable, which dynamically populates the matrix group valued
* Currently, this is failing as it treats the YML as invalid.